worker/sample.go

```diff
@@ -0,0 +1,141 @@
+package main
+
+import (
+	"bytes"
+	"encoding/csv"
+	"encoding/json"
+	"errors"
+	log "github.com/sirupsen/logrus"
+	"io/ioutil"
+	"net/http"
+)
+
+type Sample struct {
+	SAMPLEUNITREF string
```

Comment:
I look forward to the day we can cut half these fields out 😓

Reply:
I have no idea what half of them are for tbh